### PR TITLE
Change in README.md

### PR DESCRIPTION
**Pull request by lahirusamith**

### CODE REVIEW

File: java.jdbc-ballerina/build.gradle [hunk 1]
```diff
@@ -290,7 +299,7 @@ publishing {
     repositories {
         maven {
             name = "GitHubPackages"
-            url = uri("https://maven.pkg.github.com/ballerina-platform/module-ballerina-java.jdbc")
+            url = uri("https://maven.pkg.github.com/ballerina-platform/module-${packageOrg}-${packageName}")
```
Comment (daneshk) [inline java.jdbc-ballerina/build.gradle:302]: we can't use `packageOrg` here, because it refers to `ballerinax`

Comment (lahirusamith) [reply java.jdbc-ballerina/build.gradle:302]: Will do the change!

File: java.jdbc-ballerina/build.gradle [hunk 2]
```diff
@@ -262,19 +242,15 @@ task ballerinaPush {
                 workingDir project.projectDir
                 environment "JAVA_OPTS", "-DBALLERINA_DEV_COMPILE_BALLERINA_ORG=true"
                 if (Os.isFamily(Os.FAMILY_WINDOWS)) {
-                    commandLine 'cmd', '/c', "$distributionBinPath/bal.bat push && exit %%ERRORLEVEL%%"
+                    commandLine 'cmd', '/c', "$distributionBinPath/bal.bat push ${packageName} && exit %%ERRORLEVEL%%"
```
Comment (BuddhiWathsala) [inline java.jdbc-ballerina/build.gradle:245]:
```suggestion
                    commandLine 'cmd', '/c', "$distributionBinPath/bal.bat push && exit %%ERRORLEVEL%%"
```

File: java.jdbc-ballerina/build.gradle [hunk 3]
```diff
@@ -262,19 +242,15 @@ task ballerinaPush {
...
                 } else {
-                    commandLine 'sh', '-c', "$distributionBinPath/bal push"
+                    commandLine 'sh', '-c', "$distributionBinPath/bal push ${packageName}"
```
Comment (BuddhiWathsala) [inline java.jdbc-ballerina/build.gradle:247]:
```suggestion
                    commandLine 'sh', '-c', "$distributionBinPath/bal push"
```

**Review by BuddhiWathsala — CHANGES_REQUESTED** [review]

**Review by daneshk — APPROVED** [review]

**Review by BuddhiWathsala — APPROVED** [review]

**Pull request merged**